### DISCUSSION TITLE
Package sharing without `main/module` field by `ModuleFederationPlugin`

### DISCUSSION THREAD

**Discussion by BEGEMOT9I** [webpack/webpack#16395] [Q&A]
Hi! I want to answer about `ModuleFederationPlugin`.
I have a package (`@scope/package`), that doesn't contain `main` or `module` field in `package.json` (the reason: I have a different functionality that splitted internal) and use a utility from that (`import { util } from '@scope/package/utils`). These utlis are exported by `export` field from `package.json`:
```json
{
  "exports": {
    "./utils": {
      "require": "./dist/utils/index.cjs",
      "import": "./dist/utils/index.js"
    }
  }
}
```
I want to share this package for remotes. So, when i try to use and share this package, the everything okay, it's became shared in host and appeared in `moduleToHandlerMapping`. But, when i try to share it in remote package, it's don't appear in the `moduleToHandlerMapping`.
And my question is: can I use the shared package with only `exports` field or I must define a `main/module` field for `WMF`?

**Comment by BEGEMOT9I** [comment]
Okay, according to https://github.com/webpack/webpack/issues/12594#issuecomment-774641540, i've found thats adding a slash to `package` name at the end must fix the problem.
Btw, @ScriptedAlchemy WDYT about adding this tip to notes or docs?

**Comment by alexander-akait** [comment]
Please use the issue template, otherwise it will be converted to dicussion, thank you (if you fill out the original post using an issue template I will look)

**Reply by BEGEMOT9I** [reply]
Yeah, it might be just a discussion)
